windows/runner/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase